pkg/spec/resources_fix.go

```diff
@@ -124,23 +85,23 @@ func fixSecrets(secrets []SecretMod, appName string) ([]SecretMod, error) {
 	} else if len(secrets) > 1 {
 		for i, sec := range secrets {
 			if sec.Name == "" {
-				return nil, fmt.Errorf("name not specified for app.secrets[%d]", i)
+				return nil, fmt.Errorf("name not specified for deployment.secrets[%d]", i)
```

Comment:
This will still be `app`, because of the following reasons:

- this is not tied to deployment in anyway
- user will still be considering it as app
- the error is user facing